config.json

```diff
@@ -245,6 +245,14 @@
       "plugin": "TeamRandomizer",
       "enabled": true,
       "command": "randomize"
+    },
+    {
+  "plugin": "DiscordStats",
+  "enabled": true,
+  "discordClient": "discord",
+  "database": "mysql", // Replace this with any DB you use "mongoDB, sqlite, or keep it as mysql
```

Comment:
As @werewolfboy13 said, these changes need to be made in the `config-template.json` file. However, the comments should also be removed as they will cause the config to be an invalid JSON string. Users should not have to remove comments to run SquadJS.